src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/response.tag

```diff
@@ -26,29 +26,24 @@
   <div ${isNumResponsesMax ? 'class="col-sm-3 form-inline mobile-align-left"' : 'class="col-sm-5 form-inline mobile-align-left"'}
       ${isRecipientNameHidden ?  'style="display:none"' : 'style="text-align:right"'}>
 
-    <label for="input">
-      <span data-toggle="tooltip"
-        data-placement="top"
-        title="<%= Const.Tooltips.EVALUEE_DESCRIPTION %>">
-        Evaluee
-      </span>
+    <label>
+      <select class="participantSelect middlealign<c:if test="${not response.existingResponse}"> newResponse</c:if> form-control"
+          name="<%= Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT %>-${questionWithResponses.question.qnIndx}-${response.responseIndx}"
+          ${isNumResponsesMax ? 'style="display:none;max-width:125px"' : 'style="width:275px;max-width:275px"'}
```

Comment:
You can extract out `style=""` here and only put the conditional inside, i.e. `style="${...}"`.

Reply:
Okay.